### ISSUE TITLE
Cant Match Google Chrom Versions

### ISSUE THREAD

**Issue by user1722** [microsoft/winget-cli#3618]
### Brief description of your issue

When i Search for Google Chrom with Winget list i get the following Version with a ">"
![grafik](https://github.com/microsoft/winget-cli/assets/82225899/47e2daab-173a-499a-8708-27c75c82c0fe)
But if i search for the App i get a older Version. 
Winget cant Match the numbers somehow. I have this Problem on a lot of clients.
![grafik](https://github.com/microsoft/winget-cli/assets/82225899/3530a69b-52be-43ec-9f4e-ed381dc56945)
Uninstalling und Reinstall does download me the new Version, but the same issue exists with the next Patch

### Steps to reproduce

Trying to Upgrade Google Chrom 

### Expected behavior

Winget should show me a new Version available and then install ist

### Actual behavior

Winget finds no Updates for Google.Chrom

### Environment

```shell
Windows-Paket-Manager v1.5.2201
Copyright (c) Microsoft Corporation. Alle Rechte vorbehalten.

Windows: Windows.Desktop v10.0.19045.3324
Systemarchitektur: X64
Paket: Microsoft.DesktopAppInstaller v1.20.2201.0
```

**Comment by denelon** [comment]
WinGet list is using data in the registry for installed packages and then performing queries to associate with manifests in the configured sources to determine if an upgrade is available. Some packages use a "marketing" version or another value for their display version. When you see either a "<" or a ">" symbol, it's an indication that the version installed on the system doesn't have an exact match with one of the manifests in the configured source.

**Comment by denelon** [comment]
One of our community moderators noticed an incorrect DisplayVersion in one of the manifests. It's been merged. Can you confirm?

**Comment by user1722** [comment]
Yes, the versions are matching again. And Winget is Updating Google.Chrom again and no more "<"  or ">"
Problem seems fixed.
